CHANGELOG.md

```diff
@@ -6,8 +6,31 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+:warn: **Breaking Change ahead!** :warn:
+- Replace custom `cloudogu/golang` container with official golang container (#51)
+   - the affected targets are:
+      - `build`
+      - `static-analysis-ci`
+      - `static-analysis-local`
+   - the targets `static-analysis`/`static-analysis-ci` may no longer work on your CI because of changed container references  
+      - mounting a custom `/etc/passwd` into the golang container is advised, either manually or with help of the [ces-build-lib](https://github.com/cloudogu/ces-build-lib) `mountJenkinsUser()` step:
+      ```
+        new Docker(this)
+         .image('golang:1.14.13')
+         .mountJenkinsUser()
+         .inside
+     ```
```

Comment:
I think this section belongs in the README.md and should only be referenced here. In the README.md it could even be extended for better user support.